custom_components/healthchecksio/__init__.py

```
@@ -151,46 +151,42 @@ async def update_data(self):
 
 async def check_files(hass: core.HomeAssistant) -> bool:
     """Return bool that indicates if all files are present."""
     # Verify that the user downloaded all files.
-    base = f"{hass.config.path()}/custom_components/{DOMAIN}/"
-    missing = []
+    base: str = f"{hass.config.path()}/custom_components/{DOMAIN}/"
+    missing: list[str] = []
     for file in REQUIRED_FILES:
-        fullpath = "{}{}".format(base, file)
+        fullpath = f"{base}{file}"
         if not os.path.exists(fullpath):
```

Ruff (0.11.9)

`os.path.exists()` should be replaced by `Path.exists()`

(PTH110)